tf2/include/tf2/buffer_core.h

```diff
@@ -103,6 +104,16 @@ class BufferCore
   /** \brief Clear all data */
   void clear();
 
+  /** \brief Clear a specified frame
```

Comment:
There is already a clear() function, so I think it makes sense to keep a clearFrame() to express that they both serve a similar purpose.